spec/models/manageiq/providers/hawkular/inventory/parser/middleware_manager_spec.rb

```diff
@@ -53,7 +53,8 @@
           'Enabled'        => 'true'
         }
       }
-      expect(parser.send(:parse_datasource, server, datasource, config)).to eq(parsed_datasource)
+      inventory_obj = persister.middleware_datasources.build(:ems_ref => datasource.path)
+      expect(parser.send(:parse_datasource, datasource, inventory_obj, config)).to eq(parsed_datasource)
```

Comment:
When changing specs, tried to keep changes to the minimum. But I will change this. No need to use `send`, as you say.